Client/Application/AppDelegate.swift

```diff
@@ -118,6 +118,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
         // Ecosia: start experimentation
         startExperimentation()
+        sendAppAndDeviceInfoToMobileMeasurementPlatformService()
```

Comment:
I guess an Ecosia comment explaining this line makes sense here too, since it is not part of the `start experimentation` one

Reply:
reviewed the namings for both